docs/syntax-reference.md

```diff
@@ -645,13 +645,12 @@ Terminate a persistent query. Persistent queries run continuously until they are
 | TOPKDISTINCT| `TOPKDISTINCT(col1, k)`   | Return the distinct TopK values for the given column and window |
```

Comment:
This portion of the doc is being reworked in 4.1.x - this will end up being overwritten since it's written in .md (and the new files being pushed will be .rst).